pkg/util/nativeutils/eventbus/listener.go

```diff
@@ -66,7 +65,7 @@ func NewCallbackListener(callback func(message.Message)) Listener {
 func (c *CallbackListener) Close() {
 }
 
-var ringBufferLength = 2000
+var ringBufferLength = 5000
```

Comment:
any particular reason for such an increase of `ringBufferLength`?

Reply:
Due to performance and stability improvement observed on stress testing.  However this is still experimental and under consideration. The benchmark results raises a concern that ring buffer does drop consensus-sensitive messages in favor of transaction messages simply due to exceeding `ringBufferLength` on high transaction traffic.


```
var ringBufferLength = 10000

Results:
msg="Measured TPS 75.40 (377/5)" average_block_time=8 cumulative_txs_count=2659 height=12
msg="Measured TPS 62.83 (377/6)" average_block_time=8 cumulative_txs_count=3036 height=13
msg="Measured TPS 75.40 (377/5)" average_block_time=7 cumulative_txs_count=3413 height=14
msg="Measured TPS 37.70 (377/10)" average_block_time=7 cumulative_txs_count=3790 height=15
msg="Measured TPS 34.27 (377/11)" average_block_time=8 cumulative_txs_count=4167 height=16
msg="Measured TPS 75.40 (377/5)" average_block_time=7 cumulative_txs_count=4544 height=17
msg="Measured TPS 75.40 (377/5)" average_block_time=7 cumulative_txs_count=4921 height=18
msg="Measured TPS 62.83 (377/6)" average_block_time=7 cumulative_txs_count=5298 height=19
```


```
var ringBufferLength = 2000

Results:
msg="Measured TPS 75.40 (377/5)" average_block_time=10 cumulative_txs_count=3029 height=13
msg="Measured TPS 34.27 (377/11)" average_block_time=10 cumulative_txs_count=3406 height=14
msg="Measured TPS 37.70 (377/10)" average_block_time=10 cumulative_txs_count=3783 height=15
msg="Measured TPS 17.95 (377/21)" average_block_time=11 cumulative_txs_count=4160 height=16
msg="Measured TPS 15.08 (377/25)" average_block_time=12 cumulative_txs_count=4537 height=17
msg="Measured TPS 37.70 (377/10)" average_block_time=12 cumulative_txs_count=4914 height=18
msg="Measured TPS 62.83 (377/6)" average_block_time=11 cumulative_txs_count=5291 height=19
```